### DISCUSSION TITLE
Error behaviour when session cannot be created due to "time cap" limitation

### DISCUSSION THREAD

**Discussion by jlurien** [camaraproject/QualityOnDemand#231] [General]
Use case would be that QoD is productized as number of N QoD minutes or sessions per period and after reaching that limit a new QoD session, or a session duration extension, is requested. What would be the right error to be answered? As an enhancement this can be added as example to document the expected behaviour.

Some possibilities are 403, or maybe 409, with an appropriate error code and message

**Comment by hdamker** [comment]
Addressed by #306 and #297 ... adding code 429
